src/FormattingSettingsService.ts

```diff
@@ -80,33 +81,39 @@ export class FormattingSettingsService implements IFormattingSettingsService {
                         const groupUid = cardGroupInstance.name + "-group";
 
                         // Build formatting group for each group
                         const formattingGroup: visuals.FormattingGroup = {
-                                displayName: isSimpleCard ? undefined : (this.localizationManager && cardGroupInstance.displayNameKey)
-                                    ? this.localizationManager.getDisplayName(cardGroupInstance.displayNameKey) : cardGroupInstance.displayName,
-                                description: isSimpleCard ? undefined : (this.localizationManager && cardGroupInstance.descriptionKey)
-                                    ? this.localizationManager.getDisplayName(cardGroupInstance.descriptionKey) : cardGroupInstance.description,
-                                slices: [],
-                                uid: groupUid,
-                                collapsible: cardGroupInstance.collapsible,
-                                delaySaveSlices: cardGroupInstance.delaySaveSlices,
-                                disabled: cardGroupInstance.disabled,
-                                disabledReason: cardGroupInstance.disabledReason,
-                            }
+                            displayName: isSimpleCard ? undefined : (this.localizationManager && cardGroupInstance.displayNameKey)
+                                ? this.localizationManager.getDisplayName(cardGroupInstance.displayNameKey) : cardGroupInstance.displayName,
+                            description: isSimpleCard ? undefined : (this.localizationManager && cardGroupInstance.descriptionKey)
+                                ? this.localizationManager.getDisplayName(cardGroupInstance.descriptionKey) : cardGroupInstance.description,
+                            slices: [],
+                            uid: groupUid,
+                            collapsible: cardGroupInstance.collapsible,
+                            delaySaveSlices: cardGroupInstance.delaySaveSlices,
+                            disabled: cardGroupInstance.disabled,
+                            disabledReason: cardGroupInstance.disabledReason,
+                        }
                         formattingCard.groups.push(formattingGroup);
 
                         // In case formatting model adds data points or top categories (Like when you modify specific visual category color).
                         // these categories use same object name and property name from capabilities and the generated uid will be the same for these formatting categories properties
                         // Solution => Save slice names to modify each slice uid to be unique by adding counter value to the new slice uid
                         const sliceNames: { [name: string]: number } = {};
 
+                        // creating slices using same uid as described above is ok at this stage, powerbi identify the specific slice by the selector.
```

Comment:
Please dont add and wrong ways, Since people will not know what was the main problem to create this.
So add description that we are supporting slices with same, And if selector (specify its not altConstantSelector) is different we keep the same uid for the slice, if not we have to change the uid (or else it wont work)
and add that the way when the slice name is similar and selector is different is the recommended way